src/Microsoft.Identity.Web/AzureADB2COpenIDConnectEventHandlers.cs

```diff
@@ -102,8 +102,9 @@ private string BuildIssuerAddress(RedirectContext context, string? defaultUserFl
         {
             if (!_userFlowToIssuerAddress.TryGetValue(userFlow, out var issuerAddress))
             {
-                issuerAddress = context.ProtocolMessage.IssuerAddress.ToLowerInvariant()
-                    .Replace($"/{defaultUserFlow?.ToLowerInvariant()}/", $"/{userFlow.ToLowerInvariant()}/");
+                issuerAddress = context.ProtocolMessage.IssuerAddress
+                    .Replace($"/{defaultUserFlow}/", $"/{userFlow}/", StringComparison.OrdinalIgnoreCase);
```

Comment:
would two things with different casing match in the "replace"? i didn't think Replace worked w/casing, so the switch to lower needs to happen before the replace.

Reply:
I didn't see any unit tests failing. I can take a look at adding some unit tests.

Reply:
I reverted this logic as I do not understand it / don't have context it in. Generally avoid ToLower, see https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1308